libs/sqlserver/langchain_sqlserver/vectorstores.py

```diff
@@ -1243,9 +1243,9 @@ def _insert_embeddings(
                     # For a query, if there is no corresponding ID,
                     # we generate a uuid and add it to the list of IDs to be returned.
                     if idx < len(ids):
-                        custom_id = ids[idx]
+                        custom_id = str(ids[idx]).upper()
```

Comment:
Instead of doing conversion of uuid to str, do we really need to keep the datatype of `id` column to be `varchar(1000)` ? Should we instead change the datatype to uuid? 

What are the consequences of doing so?